keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/ImageService.kt

```diff
@@ -179,20 +179,17 @@ class ImageService(
         amiMatches(allTags, buildHost, buildName, buildNumber)
       }
 
-  suspend fun findBaseAmi(baseImage: String): String {
-    return cloudDriverService.namedImages(DEFAULT_SERVICE_ACCOUNT, baseImage, "test")
+  suspend fun findBaseAmiVersion(baseImageName: String): String {
+    return cloudDriverService.namedImages(DEFAULT_SERVICE_ACCOUNT, baseImageName, "test")
       .lastOrNull()
       ?.let { namedImage ->
```

Comment:
You just simplified here, no change in logic, correct?

Reply:
Correct. Although as @luispollo pointed out on my PR yesterday I think there was a risk of the `first` method throwing an exception in the unlikely event that there was no matching element, whereas `find` will return `null`